interfaces/Portalicious/src/app/domains/project/project.helper.ts

```diff
@@ -85,16 +82,68 @@ export function projectHasPhysicalCardSupport(project?: Project) {
 }
 
 export function projectHasFspWithExportFileIntegration(project?: Project) {
```

Comment:
I decided not to look at the transactions in the end, and to just keep the behaviour the same for now.